environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/SimpleEnvironmentResponse.java

```diff
@@ -14,6 +15,8 @@
 
     private CredentialViewResponse credential;
 
+    private ProxyViewResponse proxyConfig;
```

Comment:
are we sure we need it in the Simple one?

Reply:
@doktoric What do you think?

Reply:
yes we need that.